monkey/infection_monkey/monkey.py

```diff
@@ -311,7 +317,9 @@ def _build_server_list(self, relay_port: int) -> Sequence[str]:
         return list(ordered_servers.keys())
 
     def _build_puppet(self) -> IPuppet:
-        plugin_registry = PluginRegistry(self._island_api_client)
+        create_secure_directory(TMP_DIR)
+        create_secure_directory(PLUGIN_DIR)
```

Comment:
If these aren't used anywhere else, we should just make them local variables here. Define them close to where they're used.